vscode/src/memfs.ts

```diff
@@ -41,8 +41,9 @@ function populateSamples(vfs: MemFS) {
   vfs.createDirectory(playgroundRootUri.with({ path: "/samples" }));
 
   samples.forEach((sample) => {
+    const sanitized_file_name = sample.title.replace(/\W/g, "");
```

Comment:
I can't tell if special characters like `'` or `(` or handled elsewhere or missed (some of the sample titles have them)

Reply:
The regex handles those characters. \W is anything that is not alphanumeric + underscore.

Reply:
`\W` is equivalent to `[^a-zA-Z0-9_]`

Reply:
Oh, I always think "w" stands for "whitespace" 🤦  . Maybe a code comment to help out regex noobs like me.

Reply:
added